lib/ex_machina/ecto.ex

```diff
@@ -50,7 +50,7 @@ defmodule ExMachina.Ecto do
   This is only for use with Ecto models.
 
   Will return a map with the fields and virtual fields, but without the Ecto
-  metadata and associations.
+  metadata, associations, or the primary key.
```

Comment:
I think "and" makes more sense here. Otherwise it seems like it might delete metadata, or it might delete associations or it might drop primary key. In reality it drops them all